recipes/aicsimageio/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "aicsimageio" %}
+{% set version = "4.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 478330ac9dc1cd1f0d68c258be618b23ec2c76fafd06fb8e516cf247ede6d6b2
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - pytest-runner >=5.2
+  run:
+    - python >=3.7
```

Comment:
It seems all these are needed @tlambert03 ?

https://github.com/AllenCellModeling/aicsimageio/blob/main/setup.py#L81

Reply:
yes, but only because of pytest-runner... I'm chatting with @JacksonMaxfield about this at the moment :)

Reply:
oh sorry... nevermind, I'm debugging something else!